README.md

```diff
@@ -63,3 +63,12 @@ iex> LogflareLogger.info("testing log message")
 Use the `:error_string` metadata key when logging, which is for additional information that we want to log but don't necessarily want searchable or parsed for schema updating.
 
 For example, do `Logger.error("Some error", error_string: inspect(params) )`
+
+### Seed for development
+
+On your first run with `iex -S mix phx.server`:
+```
+Logflare.Repo.insert(Logflare.Billing.Plan.changeset(%Logflare.Billing.Plan{}, %{stripe_id: "1", price: 0, period: "month", name: "Free"}))
+Logflare.Repo.insert(Logflare.Billing.Plan.changeset(%Logflare.Billing.Plan{}, %{stripe_id: "2", price: 10, period: "month", name: "Metered BYOB"}))
```

Comment:
Could we move this to the seed.exs? The script is already run on the setup alias, we can do this instead of directly inserting.

Reply:
Will move

Reply:
Just checked and we're not running the seed.exs when running ecto.setup, in fact the seed.exs has price plans that do not get inserted

Reply:
seeding is not automatically run on ecto.setup because we don't want to have the seeded plans for tests. the seeded plans are mostly for dev purposes only and are dumped from the prod db

Reply:
the seed runs on `mix setup`, which is in the dev setup instructions

Reply:
Ok now I got it sorry. Will remove from the setup.